/**:
  ros__parameters:
    hybridpath_controller:
      K1_diag: [1., 1., 5.] # First gain matrix
      K2_diag: [40.0, 30.0, 40.0] # Second gain matrix
